### DISCUSSION TITLE
DOOM3-BFG Support?

### DISCUSSION THREAD

**Discussion by Ecl1pseGST** [Garux/netradiant-custom#99] [General]
Is there any chance of future DOOM3 BFG support? RBDOOM3 BFG in particular looks beautiful and i would love to make a level for it for a project idea i have, but it seems to only work with Trenchbroom, and i would MUCH rather use Radiant.

**Comment by Garux** [comment]
What's the difference between DOOM3 and DOOM3 BFG? There is config for the former, i suppose it's easy to tune it for bfg, if any tuning is needed at all.
There is also darkradiant, focused on idtech4 specific features support. NRC idtech4 support isn't that great, it's focused on general editing convenience.

**Reply by ensiform** [reply]
BFG uses binary formats mostly. You'd probably need to talk to Robert (RB d3bfg) about how his fork of TB is going for an idea of what would be needed differently from stock game but I think it's mostly the same, however there are no tools shipped with the core BFG edition.

**Comment by Ecl1pseGST** [comment]
I don't know exactly. RBDOOM3BFG has PBR and normal map support, reflection
probes, light volumes, other changes. I asked them how i can use Radiant
and they said no you need Trenchbroom, Radiant would need tweaked to
support it. I can map for Quake 4 and regular DOOM3 just fine but i can't
access RBDOOM3BFG upgraded textures/entities/etc. I have fiddled a little
with Trenchbroom i suppose learning it wouldn't be the worst thing but till
now all i've really done is make COD Zombie maps and a Quake Live map
that's almost done, and COD being a modified RTCW engine still uses Radiant
so with some control changes NRC feels just like home.

**Comment by Garux** [comment]
If it's about porting some texture format loader, i'm in.

**Reply by ensiform** [reply]
@RobertBeckebans is the rbdoom3 author aka tr3b of xreal

https://github.com/RobertBeckebans/RBDOOM-3-BFG
